history/characters/29000_gnoll.txt

```diff
@@ -808,82 +813,47 @@
 
 #dynasty=12232
 29071={
-	name=Thug
+	name=Garfang
 	dynasty=12232
-	dna="nllolaffggl"
-	culture=gnoll religion=primitive_shamanism
-	289.4.25={ birth=yes add_trait=creature_gnoll }
-	310.1.1={ add_spouse=29078 }
-	348.2.24={ death=yes }
+	martial=10 diplomacy=12 stewardship=1 intrigue=10 learning=12
+	trait=quick
+	culture=gnoll religion=primitive_shamanism
+	disallow_random_traits = yes
+	483.5.11 = { birth=yes add_trait=creature_gnoll }
+	499.5.11 = {
+		add_trait=skilled_tactician
+		add_trait=brave
+		add_trait=lustful
+		add_trait=cruel
+		add_trait=greedy
+		add_trait=class_warrior_4
+		add_trait=wroth
+		add_trait=inspiring_leader
+		add_trait=trickster
+		add_trait=class_hunter_3
+	}
+	507.7.11 = {
+		create_bloodline = {
+			type = garfang
+		}
+	}
+	509.6.26 = {
+		death = {
+			death_reason = death_battle killer = 1
+		}
+	}
 }
 29072={
-	name=Pippin
-	dynasty=12232
-	dna="nlloelffdnl"
-	culture=gnoll religion=primitive_shamanism
-	martial=7 diplomacy=5 stewardship=5 intrigue=6 learning=5
-	trait=intricate_webweaver trait=slothful trait=cruel trait=lustful trait=depressed
-	father=29071	#Thug
-	mother=29078	#Chutzpa
-	314.11.20={ birth=yes add_trait=creature_gnoll }
-	335.1.1={ add_spouse=29079 }
-	377.5.29={ death=yes }
 }
 29073={
-	name=Orsoly
-	female=yes
-	dynasty=12232
-	dna="elloldpfggd"
-	culture=gnoll religion=primitive_shamanism
-	father=29071	#Thug
-	mother=29078	#Chutzpa
-	319.8.10={ birth=yes add_trait=creature_gnoll }
-	364.8.29={ death=yes }
 }
 29074={
-	name=Gnaage
-	female=yes
-	dynasty=12232
-	dna="nllolaffggm"
-	culture=gnoll religion=primitive_shamanism
-	father=29071	#Thug
-	mother=29078	#Chutzpa
-	321.7.1={ birth=yes add_trait=creature_gnoll }
-	386.3.15={ death=yes }
 }
 29075={
-	name=Thug
-	dynasty=12232
-	dna="nploelffdhl"
-	culture=gnoll religion=primitive_shamanism
-	martial=8 diplomacy=7 stewardship=7 intrigue=5 learning=5
-	trait=midas_touched trait=slothful trait=brave trait=charitable trait=ambitious
-	father=29072	#Pippin
-	mother=29079	#Toothy
-	341.5.18={ birth=yes add_trait=creature_gnoll }
-	389.6.1={ death=yes }
 }
 29076={
-	name=Thum
-	female=yes
-	dynasty=12232
-	dna="cppoelffdil"
-	culture=gnoll religion=primitive_shamanism
-	father=29072	#Pippin
-	mother=29079	#Toothy
-	342.2.19={ birth=yes add_trait=creature_gnoll }
-	399.7.12={ death=yes }
 }
 29077={
-	name=Marauder
-	female=yes
-	dynasty=12232
-	dna="hnlldlffddl"
-	culture=gnoll religion=primitive_shamanism
-	father=29072	#Pippin
-	mother=29079	#Toothy
-	347.10.11={ birth=yes add_trait=creature_gnoll }
-	402.1.6={ death=yes }
 }
```

Comment:
Delete IDs too if you're going to remove them.

Reply:
I was saving the ids for later when i add more characters to his dynasty.